Bug with local build

### Expected Behaviour

The test are suppose to pass on the local machine and on the build server.

### Actual Behaviour

The test are passing on the build server but not on the local machine

See output:

======================================================================
FAIL: test_form_validate_data (member.tests.FormTestCase)
Test all the step of the form with and without wrong data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/django/santropolFeast/member/tests.py", line 932, in test_form_validate_data
    self._test_referent_information_without_errors()
  File "/code/django/santropolFeast/member/tests.py", line 1140, in _test_referent_information_without_errors
    self.assertTrue(b'Required information' not in response.content)
AssertionError: False is not true

----------------------------------------------------------------------
@rjacquemin is also having a build problem with this test.

--- a/django/santropolFeast/member/tests.py
+++ b/django/santropolFeast/member/tests.py
@@ -833,8 +833,8 @@ class FormTestCase(TestCase):
 
         # test referent member is emergency member
         self.assertEqual(
-            client.client_referent.first().id,
-            client.emergency_contact.member_contact.first().id
+            client.client_referent.first().referent.id,
+            client.emergency_contact.id
         )
 
         # test_referent_name:
@@ -1127,9 +1127,10 @@ class FormTestCase(TestCase):
         self.assertTrue(b'Not a valid member' in response_error.content)
 
     def _test_referent_information_without_errors(self):
+        id = Member.objects.get(firstname="First").id
         referent_information_data = {
             "client_wizard-current_step": "referent_information",
-            "referent_information-member": "[1] First Member",
+            "referent_information-member": "[{}] First Member".format(id),
             "referent_information-firstname": "",
             "referent_information-lastname": "",
             "referent_information-work_information": "CLSC",
@@ -1157,9 +1158,10 @@ class FormTestCase(TestCase):
 
     def _test_payment_information_with_errors(self):
         # Data for the address_information step with errors.
+        id = Member.objects.get(firstname="Second").id
         payment_information_data_with_error = {
             "client_wizard-current_step": "payment_information",
-            "payment_information-member": "[2] Second Member",
+            "payment_information-member": "[{}] Second Member".format(id),
             "payment_information-firstname": "",
             "payment_information-lastname": "",
             "payment_information-billing_payment_type": "check",
@@ -1224,9 +1226,10 @@ class FormTestCase(TestCase):
 
     def _test_payment_information_without_errors(self):
         # Data for the address_information step without errors.
+        id = Member.objects.get(firstname="First").id
         payment_information_data = {
             "client_wizard-current_step": "payment_information",
-            "payment_information-member": "[1] First Member",
+            "payment_information-member": "[{}] First Member".format(id),
             "payment_information-firstname": "",
             "payment_information-lastname": "",
             "payment_information-billing_payment_type": "check",
@@ -1341,9 +1344,10 @@ class FormTestCase(TestCase):
 
     def _test_step_emergency_contact_without_errors(self):
         # Data for the address_information step without errors.
+        id = Member.objects.get(firstname="First").id
         emergency_contact_data = {
             "client_wizard-current_step": "emergency_contact",
-            "emergency_contact-member": "[1] First Member",
+            "emergency_contact-member": "[{}] First Member".format(id),
             "emergency_contact-firstname": "Emergency",
             "emergency_contact-lastname": "User",
             "emergency_contact-contact_type": "Home phone",

--- a/django/santropolFeast/member/views.py
+++ b/django/santropolFeast/member/views.py
@@ -151,7 +151,7 @@ class ClientWizard(NamedUrlSessionWizardView):
     def save_emergency_contact(self, billing_member):
         emergency_contact = self.form_dict['emergency_contact']
         e_emergency_member = emergency_contact.cleaned_data.get('member')
-        if self.payment_is_emergency_contact():
+        if self.billing_member_is_emergency_contact():
             emergency = billing_member
         elif e_emergency_member:
             e_emergency_member_id = e_emergency_member.split(' ')[0]\
@@ -252,7 +252,7 @@ class ClientWizard(NamedUrlSessionWizardView):
             return True
         return False
 
-    def payment_is_emergency_contact(self):
+    def billing_member_is_emergency_contact(self):
         emergency_contact = self.form_dict['emergency_contact']
         payment_information = self.form_dict['payment_information']
 

